jenkins-x/irsa.tf

```diff
@@ -26,7 +26,7 @@ data "aws_iam_policy_document" "tekton-bot-policy" {
 
 resource "aws_iam_policy" "tekton-bot" {
   name_prefix = "jenkins-x-tekton-bot"
-  description = "EKS tekton-bot policy for cluster ${var.cluster_id}"
+  description = "EKS tekton-bot policy for cluster ${module.eks.cluster_id}"
```

Comment:
apropos in terraform-google-jx we use _jx_ as subfolder instead of _jenkins-x_. Should we use the same name? If so which one?